client.go

```diff
@@ -90,6 +79,16 @@ func (c *Client) Send(req *http.Request, v interface{}) error {
 			if err != nil {
 				return err
 			}
+
+			if resp.StatusCode < 200 || resp.StatusCode > 299 {
```

Comment:
Not sure I understand why you moved the error handling here?  Shouldn't you check for errors before trying to decode into `v`?

Reply:
I still want to decode into v any error responses, not just within the `200` status code range.  That's why I now include `ErrorFields` int the PayoutResponse

https://github.com/gametimesf/PayPal-Go-SDK/pull/1/files#diff-e871404fd69cab40521d6b2178847cf7R314